main.py

```diff
@@ -260,6 +260,282 @@ async def on_message_edit(before, after):
+    "I": "..",
+    "J": ".---",
+    "K": "-.-",
+    "L": ".-..",
+    "M": "--",
+    "N": "-.",
+    "O": "---",
+    "P": ".--.",
+    "Q": "--.-",
+    "R": ".-.",
+    "S": "...",
+    "T": "-",
+    "U": "..-",
+    "V": "...-",
+    "W": ".--",
+    "X": "-..-",
+    "Y": "-.--",
+    "Z": "--..",
+    "1": ".----",
+    "2": "..---",
+    "3": "...--",
+    "4": "....-",
+    "5": ".....",
+    "6": "-....",
+    "7": "--...",
+    "8": "---..",
+    "9": "----.",
+    "0": "-----",
+    ", ": "--..--",
+    ".": ".-.-.-",
+    "?": "..--..",
+    "/": "-..-.",
+    "-": "-....-",
+    "(": "-.--.",
+    ")": "-.--.-",
+    "!": "-.-.--",
+    ",": "--..--",
+}
+
+# we make a list of what to replace with what
+
+
+# Function to encrypt the string
+# according to the morse code chart
+def encrypt(message):
+    cipher = ""
+    for letter in message:
+        if letter != " ":
+
+            # Looks up the dictionary and adds the
+            # correspponding morse code
+            # along with a space to separate
+            # morse codes for different characters
+            cipher += MORSE_CODE_DICT[letter] + " "
+        else:
+            # 1 space indicates different characters
+            # and 2 indicates different words
+            cipher += " "
+
+    return cipher
+
+
+# Function to decrypt the string
+# from morse to english
+def decrypt(message):
+
+    # extra space added at the end to access the
+    # last morse code
+    message += " "
+
+    decipher = ""
+    citext = ""
+    for letter in message:
+
+        # checks for space
+        if letter != " ":
+
+            # counter to keep track of space
+            i = 0
+
+            # storing morse code of a single character
+            citext += letter
+
+        # in case of space
+        else:
+            # if i = 1 that indicates a new character
+            i += 1
+
+            # if i = 2 that indicates a new word
+            if i == 2:
+
+                # adding space to separate words
+                decipher += " "
+            else:
+
+                # accessing the keys using their values (reverse of encryption)
+                decipher += list(MORSE_CODE_DICT.keys())[list(
+                    MORSE_CODE_DICT.values()).index(citext)]
+                citext = ""
+
+    return decipher
+
+
+@bot.message_command(name="Encrypt to Morse")
+async def _tomorse(ctx, message: discord.message):
+    result = encrypt(message.content.upper())
+    await ctx.send(result)
+
+
+@bot.message_command(name="Decrypt Morse")
+async def _frommorse(ctx, message: discord.message):
+    result = decrypt(message.content)
+    await ctx.send(result)
+
+
+@bot.message_command(name="Decrypt binary")
+async def _frombinary(ctx, message: discord.message):
+
+	if message.content.lower() == "01000000 01100101 01110110 01100101 01110010 01111001 01101111 01101110 01100101":
+		await ctx.respond("SMH. Allowed mentions are turned off. go do something better.")
```

Comment:
This is childish, and doesn't even work as intended as someone could easily add another character to bypass this.

Reply:
Oh yeah, I forgot to remove that Easter Egg in the PR, besides, allowed mentions are turned off so it wouldn't actually be Bypassed

Reply:
That's part of my point. There isn't any need for this so why have it.